gpytorch/lazy/lazy_evaluated_kernel_tensor.py

```diff
@@ -176,36 +188,28 @@ def _getitem(self, row_index, col_index, *batch_indices):
         # We're going to handle multi-batch indexing with a try-catch loop
         # This way - in the default case, we can avoid doing expansions of x1 which can be timely
         except IndexError:
-            if any(not isinstance(bi, slice) for bi in batch_indices):
-                raise RuntimeError(
-                    "Attempting to tensor index a non-batch matrix's batch dimensions. "
-                    f"Got batch index {batch_indices} but my shape was {self.shape}"
-                )
-            x1 = x1.expand(*([1] * (len(batch_indices) - self.x1.dim() + 2)), *self.x1.shape)
+            x1 = x1.expand(*batch_shape, *x1.shape[-2:])
             x1 = x1[(*batch_indices, row_index, dim_index)]
 
         # Call x2[*batch_indices, col_index, :]
         try:
             x2 = x2[(*batch_indices, col_index, dim_index)]
         # We're going to handle multi-batch indexing with a try-catch loop
-        # This way - in the default case, we can avoid doing expansions of x1 which can be timely
+        # This way - in the default case, we can avoid doing expansions of x2 which can be timely
```

Comment:
What does "timely" mean in this context?

Reply:
Hmm I figure it's correct after thinking through it, but my brain always wants to read "costly in terms of time" rather than "timely". Maybe it's just my ESL...